javatests/arcs/core/data/ParticleSpecTest.kt

```diff
@@ -8,6 +8,27 @@ import org.junit.runners.JUnit4
 
 @RunWith(JUnit4::class)
 class ParticleSpecTest {
+  @Test
+  fun constructor_defaults_noClaims() {
+    val spec = ParticleSpec("FooSpec", mapOf(), "path.to.Foo")
+
+    assertThat(spec.claims).isEmpty()
+  }
+
+  @Test
+  fun constructor_defaults_noChecks() {
+    val spec = ParticleSpec("FooSpec", mapOf(), "path.to.Foo")
+
+    assertThat(spec.checks).isEmpty()
+  }
+
+  @Test
+  fun constructor_defaults_noAnnotations() {
+    val spec = ParticleSpec("FooSpec", mapOf(), "path.to.Foo")
+
+    assertThat(spec.annotations).isEmpty()
+  }
```

Comment:
Question for the reviewer: Are these sufficient tests for default constructor args?

Reply:
what about non-default ctor arguments, like connections? should we have tests to test them as well?

Reply:
It looks like the non-default constructor args aren't used anywhere in the ParticleSpec class. These are data fields in the data class, which in previous discussions, I think we talked about not needing to test.